tests/dats/datsc_connected.py

```diff
@@ -132,9 +140,10 @@ def test_secure_connection(self) -> bool:
             print(new_text, end="")
 
             # wait for pairing process to go through and see if it passed or failed
-            if "Pairing failed" in self.console_output:
-                print("Pairing failed")
-                return False
+			# Any past failures will cause this iteration to fail
+            # if "Pairing failed" in self.console_output:
+            #     print(self.name + " TEST: Pairing failed")
+            #     return False
```

Comment:
Perhaps this can be kept if we use `String.count()` to look for whether a new occurrence of "Pairing failed" has appeared.